tests/unit/cli/vyper_compile/test_compile_files.py

```diff
@@ -425,3 +425,56 @@ def test_archive_search_path(tmp_path_factory, make_file, chdir_tmp_path):
 
         used_dir = search_paths[-1].stem  # either dir1 or dir2
         assert output_bundle.used_search_paths == [".", "0/" + used_dir]
+
+
+def test_compile_interface_file(make_file):
+    interface = """
+@view
+@external
+def foo() -> String[1]:
+    ...
+
+@view
+@external
+def bar() -> String[1]:
+    ...
+
+@external
+def baz() -> uint8:
+    ...
+
+    """
+    file = make_file("interface.vyi", interface)
+    compile_files([file], ["ast", "annotated_ast", "interface", "external_interface", "abi"])
```

Comment:
would it make sense to use this here? https://github.com/vyperlang/vyper/pull/4290/files#diff-d428d4c971c9f7166899f6f2d3da5e17ca4478d8d371d803e735d6ee36b39d30R49

so we don't have to maintain the list on 2 places